sidecar/tests/test_hf_symlinks.py

```
@@ -0,0 +1,63 @@
+import os
+
+import numpy as np
+
+from sokuji_sidecar import hf_symlinks
+
+
+def _hf_layout(tmp_path, name, payload=b"\x01\x02\x03\x04"):
+    """Mimic the HF cache: model/<name> symlinks OUT into a sibling ../blobs/."""
+    blobs = tmp_path / "blobs"; blobs.mkdir(exist_ok=True)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)